src/modules/fetchScore.js

```diff
@@ -0,0 +1,15 @@
+import 'regenerator-runtime/runtime';
+
+const fetchScore = async function (url, id, score) {
+  try {
+    const res = await fetch([url, id, score].join(''));
+    const data = await res.json();
+    const objs = await data;
+
+    return objs;
+  } catch (err) {
+    return (err);
+  }
+};
+
+export default fetchScore;
```

Comment:
- [ ] I submitted data (Name:  Amine, Score: 90000) on your Leaderboard, I clicked on the refresh button, and I couldn't find the data that I have submitted. please make sure that your Leaderboard works as expected.
- *Hint: You haven't merged the previous pull request. Maybe this is why your Leaderboard doesn't work as expected.*

Reply:
I merged my previous pull request as advised and everything seem to be Ok now